cpp/src/ray/test/simple_kv_store/simple_kv_store.cc

```diff
@@ -0,0 +1,276 @@
+// Copyright 2021 The Ray Authors.
```

Comment:
rename the file to `cpp/src/ray/test/examples/simple_kv_store.cc`? we can put other examples in this folder in the future